pymongo/database.py

```diff
@@ -840,6 +840,11 @@ def __next__(self):
 
     next = __next__
 
+    def __bool__(self):
+        raise NotImplementedError("Database objects do not implement truth "
+                                  "value testing or bool(). Please compare "
+                                  "with None instead: collection is not None")
```

Comment:
"collection is not None" -> "database is not None"

Reply:
Done.